plugins/jabra-file/fu-jabra-file.rs

```diff
@@ -0,0 +1,23 @@
+// Copyright 2024 Richard Hughes <richard@hughsie.com>
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+
+#[repr(u8)]
+enum FuJabraFileCmdCmd {
+    Unknown02 = 0x02,
+    Unknown03 = 0x03,
+    Unknown07 = 0x07,
+    Unknown26 = 0x26,
+}
+
+#[derive(New)]
+struct FuJabraFileCmd {
```

Comment:
I think we need a new name for this: `fu_jabra_file_cmd_set_cmd` and `FuJabraFileCmdCmd` just looks *weird*.

Reply:
I went for packet, hope that fits.